great_expectations/data_context/store/tuple_store_backend.py

```diff
@@ -316,7 +316,8 @@ def _get(self, key):
 
     @override
     def _get_all(self) -> list[Any]:
-        raise NotImplementedError
+        keys = [key for key in self.list_keys() if key != StoreBackend.STORE_BACKEND_ID_KEY]
```

Comment:
Feels weird to have to exclude this myself. Is there a better option?

Reply:
https://app.codecov.io/gh/great-expectations/great_expectations/pull/9687/blob/great_expectations/data_context/store/tuple_store_backend.py